src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -230,6 +238,56 @@ private function updateGetOperation(\ArrayObject $pathOperation, array $mimeType
     }
 
     /**
+     * @param string $resourceClass
+     * @param string $operationName
+     *
+     * @return array
+     */
+    protected function getPaginationParameters(string $resourceClass, string $operationName)
```

Comment:
What is your evaluation criteria if a docblock is useful or not? Missing description?